### DISCUSSION TITLE
トラックボールの支持球について

### DISCUSSION THREAD

**Discussion by togepy1662** [Yowkees/keyball#202] [Q&A] [answered]
支持球のセラミックボールを1つ紛失し、予備もない状態です。
新たに購入したいと考えていますが、支持玉の大きさは2.5mmのものになるのでしょうか？
また白銀ラボでの販売はございますでしょうか？

**Comment by Yowkees** [comment, marked as answer]
セラミックボールは2mmのものになります。
単体での販売可能です。
白銀ラボウェブサイトよりメッセージをいただけますでしょうか？
価格や送付先等お話しさせてください🙏
